code/game/objects/items/devices/scanners.dm

```diff
@@ -324,9 +326,13 @@ GENE SCANNER
 
 			for(var/obj/item/organ/organ in H.internal_organs)
 				var/status = ""
-				if (organ.organ_flags & ORGAN_FAILING) status = "<font color='#cc3333'>Non-Functional</font>"
-				else if (organ.damage > organ.high_threshold) status = "<font color='#ff9933'>Severely Damaged</font>"
-				else if (organ.damage > organ.low_threshold) status = "<font color='#ffcc33'>Mildly Damaged</font>"
+				if(H.has_reagent(/datum/reagent/inverse/technetium))
+					if(organ.damage)
+						status = "<font color='#E42426'> organ is [round((organ.damage/organ.maxHealth)*100, 1)]% damaged.</font>"
+				else
+					if (organ.organ_flags & ORGAN_FAILING) status = "<font color='#cc3333'>Non-Functional</font>"
+					else if (organ.damage > organ.high_threshold) status = "<font color='#ff9933'>Severely Damaged</font>"
+					else if (organ.damage > organ.low_threshold) status = "<font color='#ffcc33'>Mildly Damaged</font>"
```

Comment:
Use proper indenting, not one line ifs.

I know you didn't write this, but might as well while you're here 😄

Reply:
Alrighty!